tools/Cargo.toml

```diff
@@ -4,10 +4,15 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+zksync_crypto = { git = "https://github.com/matter-labs/era-boojum-validator-cli", rev = "688798dc9b11e2c76c7e05b93f1ec7ff9d176a88", package = "crypto" }
```

Comment:
maybe do a branch or tag there? rather than a 'rev'?

Reply:
usually I think rev is more error resistant than branch/tag but for this use case I think that branch is fine. If anything we may want a potential breaking change to be caught before we generate the hash and add it to the file